example/d-form/README.md

```diff
@@ -16,7 +16,7 @@ let show_form ?message request =
       <p>You entered: <b><%s message %>!</b></p>
 %   end;
 
-    <%s! Dream.form_tag ~action:"/" request %>
+    <form action="/" method="post"><%s! Dream.csrf_tag request %>
```

Comment:
Why not put the CSRF field on its own line? It seems like it would be clearer to read.

Reply:
True, it was mostly that the Django code example does it that way. But I can change it pretty easily.